meilisync/source/postgres.py

```diff
@@ -88,51 +88,56 @@ def _():
 
     def _consumer(self, msg: ReplicationMessage):
         payload = json.loads(msg.payload)
-        changes = payload.get("change")
-        if not changes:
-            return
+        next_lsn = payload["nextlsn"]
+
+        changes = payload.get("change", [])
         for change in changes:
```

Comment:
I reworked quite a bit of this for neatness, hope that's okay!

The primary change was to `payload.get("change", [])` so that it doesn't early return.
I also changed `payload.get("nextlsn")` to `payload["nextlsn"]`, AFAIK there will always be a `nextlsn`.